shesmu-pluginapi/src/main/java/ca/on/oicr/gsi/shesmu/plugin/action/ActionState.java

```diff
@@ -62,10 +62,20 @@ public enum ActionState {
     this.processPriority = processPriority;
   }
 
+  /**
+   * The priority of an action state where the largest number indicates the most miserable action
```

Comment:
unclear - i thought smallest numbers were highest priority? i would want to act on the most miserable action first :(

Reply:
Now that I look, this is only used by the JIRA plugin and the replacement JIRA client won't use this. So, I say we delete this method after merging that. I'm happy to revert the comment change or we can just merge it and it goes away later. Your choice.